.pyup.yml

```diff
@@ -0,0 +1,63 @@
+# configure updates globally
+# default: all
+# allowed: all, insecure, False
+update: all
+
+# configure dependency pinning globally
+# default: True
+# allowed: True, False
+pin: True
+
+# set the default branch
+# default: empty, the default branch on GitHub
+branch: dev
+
+# update schedule
+# default: empty
+# allowed: "every day", "every week", ..
+schedule: "every day"
```

Comment:
I'd opt for something less frequently. If you [look for other repos we run](https://github.com/datasciencebr/jarbas/blob/master/.pyup.yml#L1), usually it is set to `every week`.